fastasyncworldedit/configuration/configuration.md

```diff
@@ -0,0 +1,438 @@
+    max-checks: 50000000
+    # Number of times a change can fail (e.g. if the player can't access that region)
+    max-fails: 50000000
+    # Allowed brush iterations (e.g. `//brush smooth`)
+    max-iterations: 1000
+    # Max allowed entities (e.g. cows)
+    max-entities: 1337
+    # Blockstates include Banner, Beacon, BrewingStand, Chest, CommandBlock,
+    # CreatureSpawner, Dispenser, Dropper, EndGateway, Furnace, Hopper, Jukebox,
+    # NoteBlock, Sign, Skull, Structure
+    max-blockstates: 1337
+    # Maximum size of the player's history in Megabytes:
+    #  - History on disk or memory will be deleted
+    max-history-mb: -1
+    # Maximum time in milliseconds //calc can execute
+    max-expression-ms: 50
+    # Cinematic block placement:
+    #  - Adds a delay to block placement (nanoseconds/block)
+    #  - Having an artificial delay will use more CPU/Memory
+    speed-reduction: 0
+    # Place chunks instead of individual blocks:
+    #  - Disabling this will negatively impact performance
+    #  - Only disable this for compatibility or cinematic placement
+    fast-placement: true
+    # Should WorldEdit use inventory?
+    # 0 = No inventory usage (creative)
+    # 1 = Inventory for removing and placing (freebuild)
+    # 2 = Inventory for placing (survival)
+    inventory-mode: 0
+    # Should large edits require confirmation (>16384 chunks)
+    confirm-large: true
+    # If undo and redo commands should be restricted to allowed regions
+    #  - Prevents scenarios where players can delete/reset a region, and then continue to undo/redo on it
+    restrict-history-to-regions: true
+    # List of nbt tags to strip from blocks, e.g. Items
+    strip-nbt: []
+    # If the disallowed blocks listed in worldedit-config.yml should be disallowed in all edits,
+    # not just where blocks patterns are used.
+    #  - Can prevent blocks being pasted from clipboards, etc.
+    #  - If fast-placement is disabled, this may cause edits to be slower.
+    universal-disallowed-blocks: true
+    # List of blocks to deny use of. Can be either an entire block type or a block with a specific property value.
+    # Where block properties are specified, any blockstate with the property will be disallowed (i.g. all directions
+    # of a waterlogged fence). For blocking/remapping of all occurence of a property like waterlogged, see
+    # remap-properties below.
+    # Example block property blocking:
+    #  - "minecraft:conduit[waterlogged=true]"
+    #  - "minecraft:piston[extended=false,facing=west]"
+    #  - "minecraft:wheat[age=7]"
+    disallowed-blocks:
+    - "minecraft:wheat"
+    - "minecraft:fire"
+    - "minecraft:redstone_wire"
+    # List of block properties that should be remapped if used in an edit. Entries should take the form
+    # "property_name[value1_old:value1_new,value2_old:value2_new]". For example:
+    #  - "waterlogged[true:false]"
+    #  - "age[7:4,6:4,5:4]"
+    #  - "extended[true:false]"
+    remap-properties: []
```

Comment:
I'd suggest removing the example config, we don't maintain this specific entry here.